### PR TITLE
Include "Email confirmed" in user export, resolves #1033

### LINKED ISSUE #1033: Add column (no/yes) for confirmed email address to user list export
Will be helpful when doing new list imports for mass mailing efforts.

### PR DESCRIPTION
**Pull request by dguettler**

### CODE REVIEW

**Comment by remomueller** [comment]
The change looks great, can you also add the CHANGELOG entry? Thanks!

**Comment by dguettler** [comment]
@remomueller I updated the CHANGELOG

**Comment by dguettler** [comment]
@remomueller any idea why my changes made this one test fail? I don't quite see the connection...

**Comment by remomueller** [comment]
Hi @dguettler, I reran the tests and they're passing again.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ## 24.0.0
 
 ### Enhancements
+- **General Changes**
```
Comment (remomueller) [inline CHANGELOG.md]:
```suggestion
- **Admin Changes**
```

**Pull request merged**